File: 2022-2023/sourcecode/lab/lab11/FoodyJava/app/src/main/java/pl/udu/uwr/pum/foodyjava/data/Meal.java
package pl.udu.uwr.pum.foodyjava.data;

import androidx.annotation.NonNull;
import androidx.room.Entity;
import androidx.room.PrimaryKey;

@Entity(tableName = "meals")
public class Meal {
    @PrimaryKey
    @NonNull
    public String idMeal;
    public String strMeal;
    public String strCategory;
    public String strArea;
    public String strInstructions;
    public String strMealThumb;
    public String strTags;
    public String strYoutube;
}

File: 2022-2023/sourcecode/lab/lab11/Foody/app/src/main/java/pl/udu/uwr/pum/foody/data/Meal.kt
package pl.udu.uwr.pum.foody.data

import androidx.room.Entity
import androidx.room.PrimaryKey

@Entity(tableName = "meal")
data class Meal(
    @PrimaryKey
    val idMeal: String,
    val strArea: String,
    val strCategory: String,
    val strInstructions: String,
    val strMeal: String,
    val strMealThumb: String,
    val strYoutube: String
)